server/src/resolvers/question.js

```diff
@@ -13,7 +14,13 @@ const confTagLabels = ctx =>
 
 module.exports = {
   Query: {
-    zNode: (_, args, ctx, info) => ctx.prisma.query.zNode(args, info)
+    zNode: async (_, args, ctx, info) => {
+      let node = await ctx.prisma.query.zNode(args, info)
+      if (!node.question.language) {
```

Comment:
J'ajouterai bien ça, au cas où.
```suggestion
      if (!node.question.language || !node.answer.language) {
```